python-sdk/src/astro/databases/base.py

```diff
@@ -115,7 +115,9 @@ def run_sql(
             sql = kwargs.get("sql_statement")  # type: ignore
 
         if isinstance(sql, str):
-            result = self.connection.execute(sqlalchemy.text(sql), parameters)
+            result = self.connection.execute(
+                sqlalchemy.text(sql).execution_options(autocommit=True), parameters
```

Comment:
why `.execution_options(autocommit=True)`?

Reply:
@kaxil I'm using run_sql() to run the truncate query, which needs to be executed immediately.  I can create a separate function to execute this query or move this logic to the tuncate_table() function.

Reply:
Why was it not required earlier? and why is it needed now?

Code comments on it would be good if we still need it

Reply:
what would happen if we don't do we need to call commit manually?

Reply:
@pankajastro We don't have to call it manually, but sqlalchemy calls it based on the optimization logic they have, we don't have the control over it.